### ISSUE TITLE
(URGENT) Allow explicit parentheses in encoding

### ISSUE THREAD

**Issue by manuelfuenmayor** [plurimath/plurimath#146]
There are still cases where parentheses get obviated:

Expressions:
````
- (a)_n
- (c)_n
````

Output:
![issue-2-html](https://user-images.githubusercontent.com/33627611/233704109-54a558f2-c3de-4994-8658-5d46e2533361.PNG)

Expected result:
![issue-2-original](https://user-images.githubusercontent.com/33627611/233704198-cc3c7f17-7edd-4574-a16e-1f2f4a0c1afe.PNG)

*(Issue found in https://github.com/metanorma/iso-iec-80000/pull/41)*

**Comment by ronaldtse** [comment]
The problem is that if we do semantic encoding of the parenthesis, which is for contextual grouping, they technically shouldn’t show up. We need another way of specifying explicit parenthesis that should always show up regardless of the context.

**Comment by ronaldtse** [comment]
This is how UnicodeMath handles explicit parentheses:

> To force the display of the outermost parentheses, one encloses them, in turn, within parentheses, which then become the outermost parentheses.

<img width="510" alt="Screenshot 2023-04-27 at 12 41 37 PM" src="https://user-images.githubusercontent.com/11865/234761130-b6b751d1-17b8-42a0-8285-5ce2dfad239c.png">

I think we should just use this syntax to force AsciiMath in showing parentheses.

**Comment by suleman-uzair** [comment]
Fixed in #149

**Comment by opoudjis** [comment]
> I think we should just use this syntax to force AsciiMath in showing parentheses.

It turns out asciimath.org (i.e. MathJax parser of AsciiMath) does the same thing with double parens.

We should be documenting our AsciiMath, since AsciiMath is globally underdocumented (by design). author/ref/asciimath.adoc in Metanorma.org is obsolete, and we have suppressed it. i would recommend @suleman-uzair maintain a list of decisions we have made in Plurimath, and that it be incorporated into metanorma.org, in place of author/ref/asciimath.adoc
